streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -474,6 +475,10 @@ public static StreamThread create(final TopologyMetadata topologyMetadata,
         taskManager.setMainConsumer(mainConsumer);
         referenceContainer.mainConsumer = mainConsumer;
 
+        final String stateUpdaterId = threadId.replace("-StreamThread-", "-StateUpdater-");
```

Comment:
Might be good to extract `private final static String ...` vars for both string? So far, each one only appears ones, but now that each appears twice it might be good to use a constant instead of hardcoded on two different places?